packages/expo-gl-cpp-legacy/unimodule.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "expo-gl-cpp",
+  "name": "expo-gl-cpp-legacy",
```

Comment:
Do we ever want to publish this unimodule to NPM? Would it make sense to exclude it from automatically published packages?

Reply:
yes, there is no need to publish that module, I didn't know that that publishing script supports sth like that

Reply:
Me neither, frankly speaking I just hoped it does. 😃 It looks like if we set the package to private `expotools` should filter it out 🤔 

https://github.com/expo/expo/blob/e166a8d8b72263a5ab75f04871e7a5d2da9729e9/tools/expotools/src/publish-packages/tasks/prepareParcels.ts#L37-L41